components/emoji_picker/emoji_picker.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div className="d-emoji-picker">
+    <div className="d-emoji-picker--header">
+      <emoji-tabset
+        :recently-used-tab="showRecentlyUsedTab"
+        @tabset-selected="scrollToSelectedTabset"
+      />
+    </div>
+    <div className="d-emoji-picker--body">
+      <emoji-search v-model="searchQuery" />
+      <!--      <emojiSelector -->
+      <!--        :emoji-filter="searchQuery" -->
+      <!--        :skin-tone="skinTone" -->
+      <!--        @emoji-data="updateEmojiData" -->
+      <!--        @selected-emoji="emits('selected-emoji', $event)" -->
+      <!--      /> -->
+    </div>
+    <div className="d-emoji-picker--footer">
+      <!--      <emojiData :emoji-data="emojiData" /> -->
+      <!--      <emojiSkinSelector @skin-tone="updateSkinTone" /> -->
+    </div>
+  </div>
+</template>
```

Comment:
How would someone trigger the emoji picker from the UI? do we need an anchor slot or is this component something that would be nested inside a DtPopover?

Reply:
What I imagine is they will place this on a DtPopover component.

As I understand, that's not our responsibility, we are developing the DtEmojiPicker, they can play with it as they want.
We can place it on a DtPopover on DtInput simulating a chatbox in Variants on Storybook.